pkg/option/config.go

```diff
@@ -1231,7 +1231,17 @@ func LogRegisteredOptions(entry *logrus.Entry) {
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
-		entry.Infof("  --%s='%s'", k, viper.GetString(k))
+		msg := viper.GetString(k)
+		if len(msg) > 0 {
+			entry.Infof("  --%s='%s'", k, msg)
+		} else {
+			msgs := viper.GetStringSlice(k)
+			if len(msgs) < 0 {
```

Comment:
The `len` of a slice can never be smaller than `0`. I think you can just drop this `if` clause and let the `entry.Infof` below print the key with an empty value.